.github/workflows/wkdev-sdk.yml

```diff
@@ -10,14 +10,12 @@ defaults:
   run:
     shell: bash
 jobs:
-  build:
-    runs-on: self-hosted
+  build_amd64:
+    runs-on: [self-hosted, x64]
     steps:
       - name: Set tag name
         run: |
-          if [ "${GITHUB_BASE_REF}" = 'main' ]; then
-            echo "WKDEV_SDK_TAG=latest" >> "${GITHUB_ENV}"
-          fi
+          echo "WKDEV_SDK_TAG=${GITHUB_RUN_ID}" >> "${GITHUB_ENV}"
```

Comment:
You can't override the tag like this, because the tag has meaning.

We deploy tags based on the branch name: https://github.com/Igalia/webkit-container-sdk/commit/fbf418af5e8dfe0a456b7f8fcc539cc3d11b270f